app/routes/publication-api.js

```diff
@@ -88,4 +88,29 @@ router.post('/entitlementsplans', express.json(), async (req, res) => {
   return res.json(response);
 })
 
+router.post('/readers/:user_id', express.json(), async (req, res) => {
```

Comment:
@ed-bird These two new api endpoints replace some of the data that was missing with the cutoff from Play. This one provides the base user data, whereas the other one provides more information on a particular order.

Reply:
Amazing!